### PR TITLE
feat(open_graph): add article:author & article:tag

### PR DESCRIPTION
**Pull request by SukkaW**
## What does it do?

https://github.com/hexojs/hexo/pull/3674#issuecomment-543276077 & https://github.com/hexojs/hexo/pull/3674#issuecomment-543422019

## How to test

```sh
git clone -b og-helper-tags-article https://github.com/sukkaw/hexo.git
cd hexo
npm install
npm test
```

## Pull request tasks

- [x] Add test cases for the changes.
- [x] Passed the CI test.

### CODE REVIEW

**Comment by curbengh** [comment]
`article:tag` accepts array type, but ogp [requires](https://ogp.me/#array)

> If a tag can have multiple values, just put multiple versions of the same <meta> tag on your page

e.g.
``` html
<meta property="article:tag" content="advocate">
<meta property="article:tag" content="charity">
```

ref: https://stackoverflow.com/q/36230298, yoast's [implementation](https://developer.yoast.com/code-documentation/plugins/yoast-seo/classes/wpseo_opengraph-tags/2/)

btw, just a note that this PR also deprecates `keywords`, it's mentioned in the OP's link, but it's not clear at first glance. [ref](https://yoast.com/meta-keywords/)

File: test/scripts/helpers/open_graph.js [hunk 1]
```diff
@@ -646,4 +657,36 @@ describe('open_graph', () => {
...
+  });
+
+  it('article:author - no author set', () => {
+    const result = openGraph.call({
+      page: {},
+      config: { author: undefined },
+      is_post: isPost
+    });
+
+    result.should.not.contain(meta({property: 'og:locale'}));
```
Comment (curbengh) [inline test/scripts/helpers/open_graph.js]: typo here

Comment (SukkaW) [reply test/scripts/helpers/open_graph.js]: Fixed.

File: lib/plugins/helper/open_graph.js [hunk 2]
```diff
@@ -122,6 +113,20 @@ function openGraphHelper(options = {}) {
...
+  if (author) {
+    result += og('article:author', author);
+  }
+
+  if (keywords) {
+    if (typeof keywords === 'string') keywords = keywords.split(',');
+
+    keywords.map(tag => {
+      return tag.name ? tag.name : tag;
+    }).filter(keyword => !!keyword).forEach(keyword => {
```
Comment (SukkaW) [inline lib/plugins/helper/open_graph.js]: I might change this to `filter(Boolean)` since it might be a little faster than `filter(x => !!x)`

Comment (curbengh) [reply lib/plugins/helper/open_graph.js]: More readable too.

**Pull request merged**